src/Grpc/HttpApi/test/testassets/IntegrationTestsWebsite/IntegrationTestsWebsite.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+
+    <IsPackable>false</IsPackable>
+    <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
```

Comment:
No need for this line either. If this warning ever shows up for a test asset project, we'll have hundreds of warnings.